src/Console/Formatter/ConsoleDiffer.php

```diff
@@ -17,7 +17,7 @@ public function __construct(
     ) {
         // @see https://github.com/sebastianbergmann/diff#strictunifieddiffoutputbuilder
         // @see https://github.com/sebastianbergmann/diff/compare/4.0.4...5.0.0#diff-251edf56a6344c03fa264a4926b06c2cee43c25f66192d5f39ebee912b7442dc for upgrade
-        $unifiedDiffOutputBuilder = new UnifiedDiffOutputBuilder();
+        $unifiedDiffOutputBuilder = new UnifiedDiffOutputBuilder( "--- Original\n+++ New\n", true);
```

Comment:
@staabm this may probably related of how original + new diff on Console formatter, see

https://github.com/rectorphp/rector-src/blob/d7a738c67ea10acaec04e1d49febe591aac96ed5/src/Console/Formatter/ColorConsoleDiffFormatter.php#L66-L71

Reply:
this file is the same as the one this class used by default before this PR